plans/admin.py

```diff
@@ -70,7 +72,24 @@ def queryset(self, request):
 
 
 class BillingInfoAdmin(UserLinkMixin, admin.ModelAdmin):
-    search_fields = ('user__username', 'user__email', 'tax_number', 'name')
```

Comment:
I think, instead of doing this each time in all the admin classes, you may want to make tiny function that does:

```python
def add_field_if_exists(field_name: str, admin_fields: List[str]) -> List[str]
```

Where the field name would be `email` or `username` and the `admin_fields` would be either `search_fields` or any other list field from the admin classes.

In that way you'll be doing it once.